lib/improver/tests/nbhood/vicinity/test_ProbabilityOfOccurrence.py

```diff
@@ -86,11 +86,11 @@ def setUp(self):
     def test_with_realization(self):
         """Test when a realization coordinate is present."""
         expected = np.array(
-            [[1., 1., 0.83333333, 0.66666667, 0.5],
+            [[1, 1., 0.77777778, 0.55555556, 0.33333333],
```

Comment:
I've had a quick look and I think @gavinevans is probably right and these changes are due to changes in the behaviour of neighbourhooding when we introduced padding at the edges.